app/presenters/summary/components/property.rb

```diff
@@ -0,0 +1,94 @@
+module Summary
+  module Components
+    class Property < BaseRecord
```

Comment:
This whole presenter needs modifying to be dynamic depending on whether it's res/comm/land right?